templates/chain.html

```diff
@@ -146,6 +164,82 @@ <h4>Sources</h4>
           </div>
         </div>
 
+        <!-- payloads -->
+        <div id="payloads" class="section-profile" style="display: none">
+          <div class="row">
+            <div class="column section-border" style="flex:25%;">
+              <img src="/chain/img/payload.png">
+              <h4 style="margin-bottom:-5px">Payload</h4>
+              <br>
+              <div class="toggle">
+                <label class="switch"><input type="checkbox" id="togBtnPayload" onchange="togglePayloadView()">
+                  <div class="slider round"><span class="on">ADD</span><span class="off">VIEW</span></div>
+                </label>
+                <div id="viewPayload">
+                  <p class="section-description">
+                    Lists the available payload packages that can be used to augment techniques.
+                  </p>
+                <button type="button" class="button-embedded atomic-button" onclick="payloadTableRefresh()">Refresh
+                    payload table</button>
+                </div>
+                <div>
+                  <div id="addPayload" style="display: none">
+                    <p class="section-description">Upload a new payload to the caldera server.</p>
+                    <input type="file" id="uploadPayloadFile">
```

Comment:
can we keep this styling inline: 
something like this may looks nice
https://tympanus.net/codrops/2015/09/15/styling-customizing-file-inputs-smart-way/